conjureplugin/conjureplugin.go

```diff
@@ -51,19 +52,35 @@ func Run(params ConjureProjectParams, verify bool, projectDir string, stdout io.
 			GenerateCLI:          currParam.CLI,
 			GenerateFuncsVisitor: currParam.AcceptFuncs,
 		}
+
+		conjureFilesToGenerate, err := conjure.GenerateOutputFiles(conjureDef, outputConf)
+		if err != nil {
+			return errors.Wrap(err, "failed to generate conjure output conjureFilesToGenerate")
```

Comment:
did you mean to use `errors.Wrapf()` with the `conjureFilesToGenerate` variable?

Reply:
Yes, I think that should be what is done here